### PR TITLE
refactor: styled-components conversion - partial

### PR DESCRIPTION
**Pull request by ramonlsouza**
### What does this PR do?

Converts the following components' styles to [styled-components](https://styled-components.com):
- presentation
- fullscreen-button
- status-notifier
- waiting-users/alert
- connection-status
- audio
- dropdown
- button
- modal
- toast

It also removes css-modules and node-sass dependencies, and the remaining .scss files.

This is part of the changes to replace [css-modules](https://github.com/akanix42/meteor-css-modules) dependency with a maintained alternative.

### Motivation

css-modules is no longer maintained.

### CODE REVIEW

**Comment by antobinary** [comment]
> It also removes css-modules and node-sass dependencies

\o/ :tada: :confetti_ball:

File: bigbluebutton-html5/package-lock.json [hunk 1]
```diff
@@ -4928,41 +4486,28 @@
...
-      "integrity": "sha1-F0uSaHNVNP+8es5r9TpanhtcX18="
+      "integrity": "sha1-F0uSaHNVNP+8es5r9TpanhtcX18=",
+      "dev": true
     },
     "path-key": {
       "version": "3.1.1",
       "resolved": "https://registry.npmjs.org/path-key/-/path-key-3.1.1.tgz",
-      "integrity": "sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q=="
+      "integrity": "sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q==",
+      "dev": true
```
Comment (antobinary) [inline bigbluebutton-html5/package-lock.json:4504]: Do you know what's up with the `"dev": true` all over the dependencies? I thought you moved something to a devDependencies list but I don't spot it. Maybe you are using a newer `npm` version?

Comment (ramonlsouza) [reply bigbluebutton-html5/package-lock.json:4504]: it happended when node-sass was removed... Maybe these dependencies were being used by node-sass and other dev-only package

**Comment by KDSBrowne** [comment]
@ramonlsouza, Do we still use / need  `"classnames": "^2.2.6"`?

**Comment by ramonlsouza** [comment]
> @ramonlsouza, Do we still use / need `"classnames": "^2.2.6"`?

yes, there are some places where we still use it

**Review by KDSBrowne — APPROVED** [review]

**Pull request merged**